Assets/HoloToolkit/Utilities/Scripts/RaycastHelper.cs

```diff
@@ -0,0 +1,312 @@
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+	public static class RaycastHelper
```

Comment:
There's already a class that extends raycasting.  Should this logic be there?

Reply:
The only class I can find in MRTK that adds Raycast functionality is RaycastResultComparer. This class doesn't extend raycasting, its a helper class for comparing results. Is this the class you were thinking of? I can merge into that class, but the name will be a bit misleading..